docs/help/version2-racks.md

```diff
@@ -0,0 +1,48 @@
+# Version 2 Racks
```

Comment:
I don't feel like this should be in the main help docs. Maybe in the v2 docs site and/or Console? 

I want to avoid as much confusion as possible for new folks coming in that have never heard of v2.

Reply:
Or can this information be included inline like the rest of the stuff in this doc?